src/assets/data/edition/series/1/section/5/op4/textcritics.json

```diff
@@ -1,5 +1,700 @@
+                        },
+                        {
+                            "svgGroupId": "g582",
+                            "measure": "7",
+                            "system": "Klav. o.",
+                            "position": "1. Note",
+                            "comment": "Anfang des Legatobogens versetzt von Taktanfang. Siehe <a (click)=\"ref.navigateToReportFragment({complexId: 'op4', fragmentId: 'source_A_corr'})\">Korrekturen in <strong>A</strong></a> T. 6 vorletzte Note: Bogenanfang gestrichen. <a (click)=\"ref.navigateToReportFragment({complexId: 'op4', fragmentId: 'source_A'})\"><strong>A</strong></a>: Akkoladenwechsel nach T. 6."
+                        },
+                        {
+                            "svgGroupId": "g603",
+                            "measure": "7",
+                            "system": "Ges.",
+                            "position": "6.–7. Note",
+                            "comment": "Text sic: Silbentrennung <em>que-lle</em>."
+                        },
+                        {
+                            "svgGroupId": "g634",
+                            "measure": "7",
+                            "system": "Ges.",
+                            "position": "12/8",
+                            "comment": "Text sic: Großschreibung <em>Rinnt</em> (wie <a (click)=\"ref.navigateToReportFragment({complexId: 'op4', fragmentId: 'text_George_DsR'})\"><strong>George_DsR</strong></a> am Zeilenanfang)."
+                        },
+                        {
+                            "svgGroupId": "g658",
+                            "measure": "8",
+                            "system": "Ges.",
+                            "position": "9/8",
+                            "comment": "Text sic: Großschreibung <em>Korn</em> (wie <a (click)=\"ref.navigateToReportFragment({complexId: 'op4', fragmentId: 'text_George_DsR'})\"><strong>George_DsR</strong></a> am Zeilenanfang)."
+                        },
+                        {
+                            "svgGroupId": "g614",
+                            "measure": "10",
+                            "system": "Ges.",
+                            "position": "5.–7. Note",
+                            "comment": "Text sic: Silbentrennung <em>Dä-mmer</em>."
+                        },
+                        {
+                            "svgGroupId": "g615",
+                            "measure": "11",
+                            "system": "Klav. u.",
+                            "position": "1/4",
+                            "comment": "Unterstimmenschicht: Ligaturbogen von T. 10 getilgt. Siehe <a (click)=\"ref.navigateToReportFragment({complexId: 'op4', fragmentId: 'source_A_corr'})\">Korrekturen in <strong>A</strong></a> T. 10 (8. Note): Bogenanfang rasiert. <a (click)=\"ref.navigateToReportFragment({complexId: 'op4', fragmentId: 'source_A'})\"><strong>A</strong></a>: Akkoladenwechsel nach T. 10."
```

Comment:
```suggestion
                            "comment": "Unterstimmenschicht: Ligaturbogen von T. 10 getilgt. Siehe <a (click)=\"ref.navigateToReportFragment({complexId: 'op4', fragmentId: 'source_A_corr'})\">Korrekturen in <strong>A</strong></a> T. 10 8. Note: Bogenanfang rasiert. <a (click)=\"ref.navigateToReportFragment({complexId: 'op4', fragmentId: 'source_A'})\"><strong>A</strong></a>: Akkoladenwechsel nach T. 10."
```

Reply:
Auch: Korrekturen in A T. 10 8. Note sagen: Auflöser hinzugefügt im Ges. Diese Referenz gemeint?